### ISSUE TITLE
ReEDS Buildout Page Features

### ISSUE THREAD

**Issue by WilliamsTravis** [NREL/reView#10]
Incorporate more ReEDS page functionality into the ReEDS buildout page:

1) Linked chart/graph selections
2) Variable Options
3) Graph Element options
4) Filters
5) Titles

**Comment by ppinchuk** [comment]
So are you saying the ReEDS page should have a chart plot in addition to a map plot, like the reV page?

**Comment by WilliamsTravis** [comment]
Yes, definitely. There are only so many plottable variables in the dataset we have, but they will have more land-use and other variables in the future. It's also possible to join the variables from the corresponding reV supply curve tables back to the ReEDS buildout tables.

**Comment by ppinchuk** [comment]
Yes, but we can only "join the variables from the corresponding reV supply curve tables" if the user provides them for us... will have to add that as a config option!

**Comment by WilliamsTravis** [comment]
Nah, let's just build it around whatever is plottable in the table.  Also, we know exactly where their data comes from since we build it and so can create our own datasets for demonstration. Also, there's going to be a push for incorporating more land-use data into these reeds analyses and so i expect more default fields in the near future.

**Comment by ppinchuk** [comment]
Sorry, I think I was unclear. We can only "join the variables from the corresponding reV supply curve tables" if the user provides `*the corresponding reV supply curve tables*` for us

**Comment by WilliamsTravis** [comment]
I meant that we can build our own separately from reView for now. In the future, these tables will have more fields. For now, we just need the functionality.

**Comment by ppinchuk** [comment]
Ohhhhh that makes a lot more sense. Yes, I am a huge fan of having the user curate the data themselves... it takes out much of the guesswork.

I will get those features rolling - am testing out GitHub's Projects page to see if it's helpful at all.
